htdocs/fourn/facture/card.php

```diff
@@ -2719,7 +2719,7 @@
 				}
 
 	 	 		// Reopen a standard paid invoice
-	            if (($object->type == FactureFournisseur::TYPE_STANDARD || $object->type == FactureFournisseur::TYPE_REPLACEMENT) && ($object->statut == 2 || $object->statut == 3))				// A paid invoice (partially or completely)
+	            if (($object->type == FactureFournisseur::TYPE_STANDARD || $object->type == FactureFournisseur::TYPE_REPLACEMENT || $object->type == FactureFournisseur::TYPE_CREDIT_NOTE) && ($object->statut == 2 || $object->statut == 3))				// A paid invoice (partially or completely)
```

Comment:
With 8.0, there is a full support of credit note. What happen if the credit note was consumed into another invoice, if we allow to reopen it ?

Reply:
I understand, but in 6.0 version we can't convert in future discount then i think it doesn't make any problem.

Reply:
But if you merge 6.0 in recent versions it can change current workflow...
I understand if you prefer avoid to merge this PR.